js/apps/system/_admin/aardvark/APP/frontend/js/templates/viewView.ejs

```diff
@@ -3,6 +3,7 @@
   <div class="innerContent">
     <div id="propertiesEditor" class="document-editor"></div>
     <div class="bottomButtonBar">
+      <span class="buttonBarInfo">To learn how to configure Views, please read our documentation <a href="https://docs.arangodb.com/3.4/Manual/Views/ArangoSearch/" target="_blank"><b>here</b>.</a></span>
```

Comment:
"here"-links should be avoided. Can be changed to:

`See the <a...>ArangoSearch Views documentation</a> to learn how to configure views.`

(if this isn't specific to ArangoSearch, then remove this word from the suggested link label, but also change the link to https://docs.arangodb.com/3.4/Manual/Views/)

Reply:
Thanks, will change that when GitHub is a bit more stable.